frontend_app_admin_workspace_user/build_app.sh

```diff
@@ -23,14 +23,27 @@ function logerror {
 
 
 dev=""
-if [ "$1" = "-d" ]; then
-    dev="-dev"
+only_utils=""
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -d|--development) dev=":dev" ;;
+        -u|--only-utils) only_utils="--only-utils" ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+log "creating debug file if not already exists"
+cp -u src/debug.js.sample src/debug.js
+
+if [ $only_utils != "--only-utils" ]; then
```

Comment:
```suggestion
if [ "$only_utils" != "--only-utils" ]; then
```

Quotes so the condition does not risk a syntax error depending on the value of $only_utils

Reply:
will  fix